src/loaders/svg/tvgSvgLoaderCommon.h

```diff
@@ -374,6 +374,14 @@ struct SvgCssStyleNode
 {
 };
 
+struct SvgTextNode
+{
+    char* text;
+    float x, y;
+    float fontSize;
+    char* fontFamily;
+};
```

Comment:
Ordering variables like this could be more optimal and packed in memory perspective

```cpp
char* text;
char* fontFamily;
float x, y;
float fontSize;
```